README.md

```diff
@@ -48,6 +48,18 @@ const helikaSDK = new Helika.EVENTS(api_key, gameId, EventsBaseURL.EVENTS_DEV);
 
 // Optional - if you want to disable some data from getting stored due to compliance
 helikaSDK.setDataSettings(DisableDataSettings.None);
+helikaSDK.setAppDetails({
+    platform_id: 'mySDK', //optional
+    app_version: '1.0.0',//optional
+    store_id: 'steam',//optional
+    source_id: 'facebook',//optional
+    server_version: '1.0.0', //optional, if from client server, not client app
+})
+//REQUIRED if sending userEvents, else OPTIONAL
```

Comment:
Update comment: 
If userDetails is not set, we autogenerate an anonymous id to the user. When you update the userDetails via `setUserDetails()`, we'll automatically associate the anonymous id to the user_id. 

Include any user identifying information that you'd like to keep track of such as any emails, wallet addresses, player_id, group_id, usernames, etc.